src/dnstest.erl

```diff
@@ -13,6 +13,8 @@
     #{
         question := {string() | binary(), dns:type()},
         additional => [dns:optrr()],
+        transport => udp | tcp,
+        ignore => [string() | binary()],
```

Comment:
Oh, how comes, is that ok to ignore sections that way? Shouldn't they have been passing? Also, the type is going to be `[atom()]`, that's what you used below and it's also the most sensible one.

Reply:
I think it's valid that in certain test cases it's valid to ingore some sections because you do not care to assert their contents. I also think it would make sense to add even more assert utilities such as count. Although I am also toying around with the idea of figuring out a better way to keep the expected responses e.g. storing dig outputs in files with some light templating.

Type change in [1acf084](https://github.com/dnsimple/dnstest/pull/21/commits/1acf084a9d22c6269831f0bbfe445f464fff912e)